### DISCUSSION TITLE
Insert comment on a workflow and/or single activity

### DISCUSSION THREAD

**Discussion by Banchio** [Azure/logicapps#837] [Feature Ideas/Requests]
Would be great to insert comments/description at workflow level to describe purpose of the workflow itself. This mechanism should be extended to include comments on workflows' activities as well.

**Comment by efficient-soil** [comment]
It would be extremely useful to have a description or note field, 200 chars maybe, next to each workflow in the workflow list within a Logic App. There certainly is a lot of real estate to use in that list. To be able to add a note to a disabled workflow would really help when handing over between different operation teams.
